db.sql

```diff
@@ -662,6 +685,54 @@ CREATE TABLE `interfaces` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_general_ci;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Table structure for table `inventory`
+--
+
+DROP TABLE IF EXISTS `inventory`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `inventory` (
+  `inventory_id` int(11) NOT NULL AUTO_INCREMENT,
+  `inventory_product_id` int(11) NOT NULL,
+  `inventory_location_id` int(11) NOT NULL,
+  `inventory_client_id` int(11) DEFAULT NULL,
+  `inventory_serial` varchar(200) DEFAULT NULL,
+  `inventory_barcode` varchar(200) DEFAULT NULL,
+  `inventory_vendor_id` int(11) NOT NULL,
+  `inventory_notes` text DEFAULT NULL,
+  `inventory_quantity` int(11) NOT NULL DEFAULT 0,
+  `inventory_cost` decimal(15,2) NOT NULL DEFAULT 0.00,
+  `inventory_created_at` datetime NOT NULL DEFAULT current_timestamp(),
+  `inventory_updated_at` datetime DEFAULT NULL ON UPDATE current_timestamp(),
+  `inventory_archived_at` datetime DEFAULT NULL,
+  PRIMARY KEY (`inventory_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_general_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table Structure for `inventory_locations`
+--
+DROP TABLE IF EXISTS `inventory_locations`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `inventory_locations` (
```

Comment:
Worth checking with @johnnyq but believe the table name is plural (correct) but the column names should be singular

https://docs.itflow.org/code_standards#sql_table_structure
_Column Naming: Define by non-plural word of the table name separated with an underscore. Example table users has user_id user_first_name_

Reply:
yes the field names should be singular good catch